makefile

```diff
@@ -15,3 +15,18 @@ test: vet
 	./scripts/test_all.sh
 
 .PHONY: all test
+
+changelog:
```

Comment:
`docker` is supposed to be an optional dependency? Else to use this package, we might need to install docker, which might be a big ask 😛